navapbc/wic-participant-recertification-portal
# PRP-325 & PRP-324 Manage WAF and logging resources in Terraform

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-325
https://wicmtdp.atlassian.net/browse/PRP-324

## Changes
* added waf association resources
* enable load balancer logging in terraform state
* remove kinesis data firehose and related resources

## Context for reviewers
> Associated resources for the WAF have been manually added in the AWS console. The terraform state needs to reflect this
> In the console we enabled cloudwatch logs for the ALBs and the WAF in order to make debugging and querying data easier. The terraform configuration needs to reflect what’s in the console.

## Testing
> When this PR is ready for review, the terraform state should match what is in the console
1. Navigate to the `app-waf` directory
2. run `terraform plan` or `terraform apply` and ensure that terraform won't attempt to create new resources or delete resources
-------------  ⚠️  This PR is still in draft so please don't do this yet ⚠️  ---------------------------------------------------
4. Navigate to the `app/envs/dev` directory
5. . run `terraform plan` or `terraform apply` and ensure that terraform won't attempt to create new resources or delete resources
